scripts/jenkins-yoctobuild-build.sh

```diff
@@ -418,52 +420,71 @@ if grep mender_servers <<<"$JOB_BASE_NAME"; then
-                    ./tests/build-docker -t mendersoftware/mender-client-docker:pr
-                    $WORKSPACE/integration/extra/release_tool.py --set-version-of mender-client-docker --version pr
-                fi
+                ./tests/build-docker -t $docker_url:pr
+                $WORKSPACE/integration/extra/release_tool.py --set-version-of $docker --version pr
 
-                if grep -q install-modules-gen Makefile; then
-                    make prefix=$WORKSPACE/go bindir=/bin install-modules-gen
-                fi
+                make prefix=$WORKSPACE/go bindir=/bin install-modules-gen
                 ;;
 
-            mender-api-gateway-docker)
-                cd $build
-                docker build -t mendersoftware/api-gateway:pr .
-                $WORKSPACE/integration/extra/release_tool.py --set-version-of $build --version pr
+            mender-client-qemu*)
+                # Built in build_and_test_client.
+                :
                 ;;
 
-            mender-cli)
-                cd $WORKSPACE/go/src/github.com/mendersoftware/$build
-                make install
+            api-gateway)
+                cd $git
+                docker build -t $docker_url:pr .
+                $WORKSPACE/integration/extra/release_tool.py --set-version-of $docker --version pr
                 ;;
 
             mender-conductor)
-                cd go/src/github.com/mendersoftware/$build
-                docker build -t mendersoftware/mender-conductor:pr ./server
-                if [ -f ./workers/send_email/Dockerfile ]; then
-                    docker build -t mendersoftware/email-sender:pr ./workers/send_email
-                fi
-                $WORKSPACE/integration/extra/release_tool.py --set-version-of $build --version pr
+                cd go/src/github.com/mendersoftware/$git
+                docker build -t $docker_url:pr ./server
+                $WORKSPACE/integration/extra/release_tool.py --set-version-of $docker --version pr
+                ;;
+
+            email-sender)
+                cd go/src/github.com/mendersoftware/$git
+                docker build -t $docker_url:pr ./workers/send_email
+                $WORKSPACE/integration/extra/release_tool.py --set-version-of $docker --version pr
                 ;;
 
             mender-conductor-enterprise)
-                cd go/src/github.com/mendersoftware/$build
-                docker build --build-arg REVISION=pr -t mendersoftware/mender-conductor-enterprise:pr ./server
-                docker build -t mendersoftware/org-welcome-email-preparer:pr ./workers/prepare_org_welcome_email
-                $WORKSPACE/integration/extra/release_tool.py --set-version-of $build --version pr
+                cd go/src/github.com/mendersoftware/$git
+                docker build --build-arg REVISION=pr -t $docker_url:pr ./server
+                $WORKSPACE/integration/extra/release_tool.py --set-version-of $docker --version pr
+                ;;
```

Comment:
I think this can be unified with `mender-conductor` above.

The only difference is that here we have a `--build-arg`, but the other build should just ignore it without error.

Reply:
Good point. It does produce a warning, but we can live with that!